recipes/drogon/all/patches/1.9.9-0001-find-cci-packages.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 3158fa6..cb5004b 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -398,17 +398,10 @@ endif (BUILD_POSTGRESQL)
+ 
+ if (BUILD_MYSQL)
+     # Find mysql, only mariadb client library is supported
+-    find_package(MySQL QUIET)
+-    find_package(unofficial-libmariadb QUIET)
+-    if (MySQL_FOUND)
+-        target_link_libraries(${PROJECT_NAME} PRIVATE MySQL_lib)
+-        set(DROGON_FOUND_MYSQL TRUE)
+-        set(MYSQL_LIB_NAME MySQL_lib)
+-    elseif (unofficial-libmariadb_FOUND)
+-        target_link_libraries(${PROJECT_NAME} PRIVATE unofficial::libmariadb)
+-        set(DROGON_FOUND_MYSQL TRUE)
+-        set(MYSQL_LIB_NAME unofficial::libmariadb)
+-    endif ()
++    find_package(mariadb-connector-c QUIET)
```

Comment:
This whole patch can be replaced by modern CMakeDeps usage, by setting in the drogon generate() recipe the following lines:

```
deps.set_property("mariadb-connector-c", "cmake_file_name", "MySQL")
deps.set_property("mariadb-connector-c", "cmake_target_name", "MySQL_lib")
```